### PR TITLE
Change warning modal content

### PR DESCRIPTION
**Pull request by tomheather-50**
- Change warning modal content as per updated designs 
- Removed redundant code to fetch current eth price

### CODE REVIEW

File: src/components/TransactionInformationModal/index.tsx [hunk 1]
```diff
@@ -54,35 +69,34 @@ export default function TransactionInformationModal({ onDismiss, onContinue }: C
...
           </RowBetween>
-          <RowBetween margin="2rem 0 0">
+          <RowBetween margin="0.5rem 0 0" flexDirection="column">
+            <ImageWrapper>
+              <img src={Image} />
+            </ImageWrapper>
             <Text fontWeight={300} fontSize={16}>
-              Due to a limitation with MetaMask you will see a red warning message before signing your transaction. This
-              is nothing to worry about your funds are safe.
+              mistX is completely free of Gas Fees, and uses an extra layer of security called MEV. Metamask is still
```
Comment (juice0x) [inline src/components/TransactionInformationModal/index.tsx:81]: We may need to tweak the messaging a bit, it kind of sounds like there are no fees with mistx, which isnt really true. The user still pays fees, and we aren't gasless yet afaik.

**Review by juice0x — APPROVED** [review]

File: src/components/TransactionInformationModal/index.tsx (hunk 1 above)
Comment (tomheather-50) [reply src/components/TransactionInformationModal/index.tsx:81]: @ju1cec @Dire-0x  - @CodeForcer suggested making this as short and as non-technical as possible, i tend to agree so propose we shorten this to

"Due to a limitation, Metamask will prompt a Signature request and a warning. This is an expected behaviour, please press on Sign when the popup appears"

We can explain the how mistX works in terms of fee's and technicalities elsewhere.  

Let me know your thoughts

**Review by juice0x — APPROVED** [review]
lgtm

**Review by Dire-0x — APPROVED** [review]
lgtm
